OWNERS.md

```diff
@@ -14,7 +14,10 @@ repository maintainers in their own `OWNERS.md` file.
 
 * Sergen Yalcin <sergen@upbound.io> ([sergenyalcin](https://github.com/sergenyalcin))
 * Fatih Turken <fatih@upbound.io> ([turkenf](https://github.com/turkenf)))
-* Matt Bush <mbbush@gmail.com> ([mbbush](https://github.com/mbbush))
 * Jason Tang <jasont@upbound.io> ([jastang](https://github.com/jastang))
 
 See [CODEOWNERS](./CODEOWNERS) for automatic PR assignment.
+
+## Emeritus Maintainers
```

Comment:
I kind of think @ulucinar deserves to be listed as an Emeritus Maintainer as well, but I'll leave that up to the rest of you to decide.